### PR TITLE
feat(onboarding-ui): AdminInfoPage for Cloud Registration wizard

### PR DESCRIPTION
**Pull request by dougfabris**
- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/onboarding-ui/src/forms/AdminInfoForm/AdminInfoForm.tsx [hunk 1]
```diff
@@ -37,6 +41,7 @@ const AdminInfoForm = ({
   validateUsername,
   validateEmail,
   validatePassword,
+  keepPosted,
```
Comment (tassoevan) [inline packages/onboarding-ui/src/forms/AdminInfoForm/AdminInfoForm.tsx:44]: I believe it's nice to be explicit on default values. It helps both TypeScript (not dealing with `undefined` type) and the component user (reading the default value as Storybook).

```suggestion
  keepPosted = false,
```

File: packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  validatePassword: Validate<string>;
+  onSubmit: SubmitHandler<{
+    fullname: string;
+    username: string;
+    companyEmail: string;
+    password: string;
+  }>;
+};
+
+const CloudAdminInfoPage = (props: CloudAdminInfoPageProps): ReactElement => (
```
Comment (tassoevan) [inline packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx:26]: What is different in this component in comparison with `AdminInfoPage`? It's not very clear for me.

File: packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <BackgroundLayer>
+    <FormPageLayout title={props.title} subtitle={props.subtitle}>
```
Comment (tassoevan) [inline packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx:28]: Cloud pages use the `description` prop, as far as I remember. Please check it.

File: packages/onboarding-ui/src/forms/AdminInfoForm/AdminInfoForm.tsx (hunk 1 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/forms/AdminInfoForm/AdminInfoForm.tsx:44]: Indeed

File: packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx (hunk 2 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx:26]: As we discuss previously any difference that justifies a new component, the fix will remove this one!

File: packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx (hunk 3 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/pages/CloudAdminInfoPage/CloudAdminInfoPage.tsx:28]: Hmmm ok, but should we not display the description on smaller devices?

File: packages/onboarding-ui/src/pages/AdminInfoPage/AdminInfoPage.tsx [hunk 4]
```diff
@@ -7,9 +7,12 @@ import AdminInfoForm from '../../forms/AdminInfoForm';
 import type { AdminInfoPayload } from '../../forms/AdminInfoForm/AdminInfoForm';
 
 type AdminInfoPageProps = {
+  title?: string;
+  description?: string;
```
Comment (tassoevan) [inline packages/onboarding-ui/src/pages/AdminInfoPage/AdminInfoPage.tsx:11]: For more flexibility:

```suggestion
  title?: ReactNode;
  description?: ReactNode;
```

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: packages/onboarding-ui/src/pages/AdminInfoPage/AdminInfoPage.tsx (hunk 4 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/pages/AdminInfoPage/AdminInfoPage.tsx:11]: Sounds good!

**Review by tassoevan — APPROVED** [review]

**Pull request merged**